Gemfile.rails3

```diff
@@ -0,0 +1,6 @@
+source 'https://rubygems.org'
+
+# Specify your gem's dependencies in hestia.gemspec
+gemspec
+
+gem "actionpack", ">= 3", "< 4"
```

Comment:
Maaybe. I find it easier to reason about like this, but you make a fair point. I've used the pessimistic operator in the other Gemfiles, so perhaps I should just be consistent.